docs/modules/operation/pages/bsm/business-service-map.adoc

```diff
@@ -27,6 +27,39 @@ image::bsm/02_bsm-service-hierarchy.png[]
 
 (4) Reduction key edge used as an input to the top-level business service, which references just a node-lost service of a page sequence monitor for the user login.
 
-To add or remove an additional selected BS or edge to focus use, in the context menu Add To Focus or Remove From Focus.
-If you want to have a specific BS or edge as a single focus use Set as Focal Point.
+To add or remove an additional selected business service  or edge to focus use, in the context menu Add To Focus or Remove From Focus.
```

Comment:
I'm finding it difficult to locate these options or the context menu. Is there some way to be more specific?

Reply:
The options only appear if the action is applicable to the item you are right-clicking on.